### PR TITLE
[FLINK-35819][Connectors/AWS] Update to AWS SDKv2 2.26.19

### PR DESCRIPTION
**Pull request by karubian**
## Purpose of the change

Update to AWS SDKv2 2.26.19

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Significant changes
- [X] Dependencies have been added or upgraded
- [ ] Public API has been changed (Public API is any class annotated with `@Public(Evolving)`)
- [ ] Serializers have been changed
- [ ] New feature has been introduced
  - If yes, how is this documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Review by iemre — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@ under the License.
 
     <properties>
         <aws.sdkv1.version>1.12.439</aws.sdkv1.version>
-        <aws.sdkv2.version>2.20.144</aws.sdkv2.version>
+        <aws.sdkv2.version>2.26.19</aws.sdkv2.version>
```
Comment (hlteoh37) [inline pom.xml:57]: Do we need to update any `NOTICE` files?

Comment (z3d1k) [reply pom.xml:57]: Yes, we need to also update all mentions of the SDK versions in `NOTICE` files

**Review by z3d1k — CHANGES_REQUESTED** [review]
Thank you for the contribution @karubian, please also update version in `NOTICE` files

File: pom.xml (hunk 1 above)
Comment (karubian) [reply pom.xml:57]: Thank you, updated

**Review by z3d1k — APPROVED** [review]
LGTM

**Review by hlteoh37 — APPROVED** [review]
LGTM - Thanks @karubian :D

**Pull request merged**